README.adoc

```diff
@@ -115,6 +114,8 @@ The MicroProfile Metrics API was added as a dependency to your `pom.xml` file. L
 the `org.eclipse.microprofile.metric` group ID. Add this dependency to use MicroProfile Metrics API
 in your code to provide metrics from your microservices.
 
+Navigate to the `start` directory.
```

Comment:
This is not consistent with the similar changes applied on the other guide, ie, `Before you begin, navigate to the `start` directory.` and to be added at the very beginning of the section.

Reply:
Just updated it.